modules/acs-quick-install-secured-cluster-using-helm.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_helm/install-helm-quick.adoc
+:_module-type: PROCEDURE
+[id="installing-secured-cluster-services-quickly_{context}"]
+= Installing the secured-cluster-services Helm chart without customization
+
+[role="_abstract"]
+Use the following instructions to install the secured-cluster-services Helm chart to deploy the per-cluster and per-node components (Sensor, Admission Controller, and Collector).
+
+[CAUTION]
+====
+To install Collector on systems configured with Unified Extensible Firmware Interface (UEFI) boot, you must use eBPF probes because kernel modules are unsigned, and the UEFI firmware cannot load unsigned packages.
+====
+
+.Prerequisites
+* You must have the address and the port number on which you are exposing the Central service.
```

Comment:
Not critical, but suggest "... port number that you are exposing the Central service on."